solidity/contracts/converter/LiquidTokenConverter.sol

```diff
@@ -127,7 +127,7 @@ contract LiquidTokenConverter is BancorConverter {
             return sell(_amount, _beneficiary);
 
         // invalid input
-        revert();
+        revert("BANCOR_ERR_INVALID_TOKEN");

```

Comment:
@yudilevi : See previous comment (in `BancorFormula.sol`) about revert messages in read-only functions.